nlfyapp/src/components/button.js

```diff
@@ -11,7 +11,7 @@ const PressableButton = styled(Pressable)`
   padding-vertical: 16px;
   padding-horizontal: 32px;
   border-radius: 50px;
-  top: 50px;
+  top: 0px;
```

Comment:
Remove the top totally since 0px is equivalent of not having any top.

Reply:
I have removed the top, please review